db/migrations/setting_up_db.sql

```diff
@@ -0,0 +1,20 @@
+CREATE DATABASE chitter;
+
+\c chitter
+
+Create Users table
+
+CREATE TABLE users(id SERIAL PRIMARY KEY, username VARCHAR(60), message_id INT);
+CREATE TABLE
```

Comment:
Are these second CREATE TABLE's typos? Obviously time was short but perhaps doing a test run of deleting your databases and then following these steps (or even better, using a rakefile) would eliminate this?